File: composeApp/src/desktopMain/java/ies/java/model/Marca.java
package ies.java.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "marca")
public class Marca {
    @Id
    @Column(name = "marca", nullable = false, length = 15)
    private String marca;

    @Column(name = "empresa", length = 60)
    private String empresa;

    @Column(name = "logo")
    private byte[] logo;

    public String getMarca() {
        return marca;
    }

    public void setMarca(String marca) {
        this.marca = marca;
    }

    public String getEmpresa() {
        return empresa;
    }

    public void setEmpresa(String empresa) {
        this.empresa = empresa;
    }

    public byte[] getLogo() {
        return logo;
    }

    public void setLogo(byte[] logo) {
        this.logo = logo;
    }

}

File: composeApp/src/desktopMain/java/ies/java/model/Marca.kt
package ies.java.model

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "marca")
open class Marca {
    @Id
    @Column(name = "marca", nullable = false, length = 15)
    open var marca: String? = null

    @Column(name = "empresa", length = 60)
    open var empresa: String? = null

    @Column(name = "logo")
    open var logo: ByteArray? = null
}